README.md

```diff
@@ -18,12 +18,13 @@ A high precision scientific calculator with full support for physical units.
 Contents
 --------
 
--   [Documentation](#documentation)
--   [Reference](#reference)
--   [Pros and cons](#pros-and-cons)
--   [FAQ](#faq)
--   [Terminal version](#terminal-version)
--   [Development](#development)
+- [Contents](#contents)
```

Comment:
Could you please manually reset these changes in the README? I guess they are caused by a different/new pandoc version?

Reply:
I thought "why does this keep failing" until I realized pandoc is just that fast 😄.
Unfortunately, it spits out a different TOC. Anyway, it's back to normal now.